File: Python-Solutions/Problem1.py
#Problem 1 - Multiples of 3 or 5

NUMBER = 1000

def multipleOfThreeOrFive(number):
    if number%3==0 or number%5==0:
        return True
    return False


sum = 0
for i in range(1,NUMBER):
    if multipleOfThreeOrFive(i) == True:
        sum+=i

print(f"The sum is: {sum}")

File: Java-Solutions/Problem1.java
// Problem 1 - Multiples of 3 or 5

public class Problem1 {
    public static void main(String[] args) {
        final int NUMBER = 1000;

        long sum = 0;
        
        for (int i = 2; i<NUMBER; i++) {
            if (i%3==0 || i%5==0) {
                sum+=i;
            }
        }

        System.out.println("The sum is: " + sum);
    }
}